python/lsst/ts/m2/model.py

```diff
@@ -0,0 +1,849 @@
+        zenith_angle : `float`
+            Zenith angle in degree.
+        n_actuators : `int`
+            Number of axial actuators.
+        n_tangent_actuators : `int`
+            Number of tangent actuators.
+        mirror_position : `dict`
+            Mirror position. The key is the axis (x, y, z, xRot, yRot, zRot) of
+            mirror. The units are the micron and arcsec.
+        temperature : `dict`
+            Temperature of mirror in degree C.
+        axial_forces : `dict`
+            Forces of the axial actuators in Newton.
+        tangent_forces : `dict`
+            Forces of the tangent actuators in Newton.
+        force_balance : `dict`
+            Force balance system contains the information of force and moment.
+            The units are the Newton and Newton * meter.
+        force_balance_system_status : `bool`
+            Force balance system is on or off.
+        lut : `dict`
+            Look-up table (LUT).
+        error_cleared : `bool`
+            Error is cleared or not.
+        """
+
+        # Set the logger
+        if log is None:
+            self.log = logging.getLogger(type(self).__name__)
+        else:
+            self.log = log.getChild(type(self).__name__)
+
+        self.telemetry_interval = telemetry_interval
+
+        self.inclination_source = MTM2.InclinationTelemetrySource.ONBOARD
+        self.zenith_angle = 0.0
+
+        self.n_actuators = 72
+        self.n_tangent_actuators = 6
+
+        self.mirror_position = dict(
+            [(axis, 0.0) for axis in ("x", "y", "z", "xRot", "yRot", "zRot")]
+        )
+
+        self.temperature = self._get_default_temperature()
+
+        self.axial_forces = dict(
+            [
+                (item, np.zeros(self.n_actuators))
+                for item in (
+                    "lutGravity",
+                    "lutTemperature",
+                    "applied",
+                    "measured",
+                    "hardpointCorrection",
+                )
+            ]
+        )
+        self.tangent_forces = dict(
+            [
+                (item, np.zeros(self.n_tangent_actuators))
+                for item in (
+                    "lutGravity",
+                    "lutTemperature",
+                    "applied",
+                    "measured",
+                    "hardpointCorrection",
+                )
+            ]
+        )
+        self.force_balance = dict(
+            [(axis, 0) for axis in ("fx", "fy", "fz", "mx", "my", "mz")]
+        )
+        self.force_balance_system_status = False
+
+        self.lut = dict()
+
+        self.error_cleared = True
+
+    def _get_default_temperature(self, init_temperature=11.0):
+        """Get the default temperature. The unit is degree C.
+
+        Parameters
+        ----------
+        init_temperature : `float`, optional
+            Initial temperature in degree C. (default is 11.0)
+
+        Returns
+        -------
+        `dict`
```

Comment:
Usually when you return a named variable, you can add the name to the docstring return section. It helps developers to know which variable will be returned, e.g.;

```
Returns
--------
temperature : `dict`
```

Reply:
Done in the commit: f20e3d7. Thanks!